IndicoV2.Abstractions/Workflows/Models/Workflow.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using IndicoV2.StrawberryShake;
+
+namespace IndicoV2.Workflows.Models
+{
+    public class WorkflowSs : IWorkflow
+    {
+        private readonly IListWorkflows_Workflows_Workflows _ssWorkflow;
+        public WorkflowSs(IListWorkflows_Workflows_Workflows workflow) => _ssWorkflow = workflow;
+        public int Id => _ssWorkflow.Id ?? 0;
```

Comment:
```suggestion
    //this allows you to use the property style object construction / the = 0; is called default auto-property syntax 
        public int Id {get; set;} = 0;
```

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/how-to-initialize-objects-by-using-an-object-initializer